extensions/ql-vscode/src/queries-panel/queries-panel.ts

```diff
@@ -14,4 +20,19 @@ export class QueriesPanel extends DisposableObject {
     });
     this.push(treeView);
   }
+
+  public getCommands(): QueriesPanelCommands {
+    return {
+      "codeQLQueries.runLocalQueryContextInline": this.runLocalQuery.bind(this),
+    };
+  }
+
+  private async runLocalQuery(
+    queryTreeViewItem: QueryTreeViewItem,
+  ): Promise<void> {
+    await this.app.queryServerCommands.execute(
+      "codeQLQueries.runLocalQueryFromQueriesPanel",
+      vscode.Uri.file(queryTreeViewItem.path),
```

Comment:
My preference would be to import `Uri` from `vscode` directly, so that we could call this instead:

```suggestion
     Uri.file(queryTreeViewItem.path),
```

This is how we're doing this in most other files as well.

Reply:
👍 I used vscode here bc line 18, but adjusted both now